src/sinks/mod.rs

```diff
@@ -298,10 +298,15 @@ pub enum Sinks {
     #[configurable(metadata(docs::label = "Kafka"))]
     Kafka(kafka::KafkaSinkConfig),
 
+    /// Mezmo (formerly LogDNA).
```

Comment:
```suggestion
    /// Deliver log event data to Mezmo.
```

It's hard to tell when this rename happened, but from where this gets used in the UI I think a more succinct description would be better. Alternatively leave the "formerly" bit in but still include the "Deliver..." portion.